ios/edge/AppDelegate.m

```diff
@@ -53,6 +54,13 @@ - (BOOL)application:(UIApplication *)application
   // Client-side background fetch interval:
   [[UIApplication sharedApplication] setMinimumBackgroundFetchInterval:60*60*12];
 
+  // Notification permissions:
+  [[UNUserNotificationCenter currentNotificationCenter]
+      requestAuthorizationWithOptions:UNAuthorizationOptionBadge | UNAuthorizationOptionAlert
+      completionHandler: ^(BOOL granted, NSError *error) {
+    printf("boom");
```

Comment:
Would you prefer to change this is something a bit more useful?

Reply:
Oops, leftover debugging code...